activity-metrics-list.md

```diff
@@ -28,6 +28,7 @@ We list and describe activity metrics. For different situations, the metrics hav
 |Dependency Depth|*Number of projects included in code base + number of projects relying on focal project (recursive)*\\ Indicator about centrality in open source Dependency network|
 |Distribution of Work|How much recent activity is distributed?|
 |Downloads|*Number of downloads*\\ ! beware: downloads might be skewed by builders\\ Used as measure for 'success' (Grewal, Lilien, & Mallapragada, 2006) |
+|Elephant Factor|If 50% of community members are employed by the same company, it is the elephant in the room. Formally: The minimum number of companies whose employees perform 50% of the commits|

```

Comment:
You want to clarify whether it's 50% of community members that are employed by the same company or 50% of contributions. That can vary widely. IMHO, it should be the latter.

I'm not sure I understand the current formal description of the metric, is it: 

> The percentage of contributions made by the largest contributing company?

So a project where 400 out of 500 contributions were made by Example, Inc. employees would have an elephant factor of 0.8 (or 80%). Is that a correct interpretation?

Reply:
@tobie, I agree. I think it should be the latter. Contributions is the key and tells much more of the story. 

I think that your assessment, 400 out of 500, is correct.

Reply:
I understand the definition differently: (1) Select the organization with the most commits, (2) only if selected organizations have less than 50% of all commits, add the remaining organization with the most commits, (3) if statement in 2 evaluates to false, count the number of selected organizations.

In your example (400/500 by one company), the Elephant Factor is 1 because this one company has 50+% of all commits.

Reply:
Regardless of which definition we decide on. (although, maybe we should go with a definition found elsewhere), the information has similar value once the most dominant company has less than 50%.

https://opensource.com/business/15/12/top-5-open-source-community-metrics-track

Reply:
I think I was one of those presenting the Elephant Factor for the first time. In our definition, it was "minimum number of companies needed to reach 50% of commits, when adding up commits for their employees working in the project". As you comment, activity is what matters. This is copycat from the "Pony Factor", by Apache, which is defined the same way, but for a single developer.

Reply:
That makes more sense now, @jgbarah. We want to align the definitions in that case, imho.